dev-docs/references/directory_layout.md

```diff
@@ -17,4 +17,15 @@ The following describes the intent of UA client related directories:
 | ./apt-conf.d/ | apt config files delivered to /etc/apt/apt-conf.d to automatically allow unattended upgrades of ESM security-related components. If apt proxy settings are configured, an additional apt config file will be placed here to configure the apt proxy. |
 | /etc/ubuntu-advantage/uaclient.conf | Configuration file for the UA client.|
 | /var/lib/ubuntu-advantage/private | `root` read-only directory containing Contract API responses, machine-tokens and service credentials |
+| /var/lib/ubuntu-advantage/machine-token.json | `world` readable directory containing redacted Contract API responses, machine-tokens and service credentials |
```

Comment:
I think we should change `directory` to `file` here

Reply:
Oh yeah, updating.